s4e-backend/src/main/java/pl/cyfronet/s4e/ex/ErrorHandler.java

```diff
@@ -65,6 +65,26 @@
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
     }
 
+    @ExceptionHandler(InstitutionCreationException.class)
```

Comment:
Wouldn't these exceptions map to MethodArgumentNotValidException?
The exception occurred because some field (name) was invalid in some way, so maybe it would be worth to return a description of it?

Reply:
added description with old ex